### PR TITLE
Stanislav Hryhorskiy Added homework 7

### PR DESCRIPTION
**Pull request by Stanislav-Hryhorskyi**
I am waiting for the review code)

### CODE REVIEW

File: Stanislav_Hryhorskiy/HW1/hw_1_practical_task_1.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+WRITE A PROGRAM THAT WILL DISPLAY THE FOLLOWING QUESTIONS FOR USER:
+      “WHAT IS YOUR NAME?“
+       “HOW OLD ARE YOU?“
+       “WHERE DO YOU LIVE?“
+and will read the user's responses and display the corresponding messages:
+      “HELLO, (ANSWER(NAME))“.
+      “YOUR AGE IS  (ANSWER(AGE))“.
+      “YOU LIVE IN  (ANSWER(CITY))“.
+"""
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW1/hw_1_practical_task_1.py:12]: good job

File: Stanislav_Hryhorskiy/HW1/hw_1_practical_task_2.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+AND WILL OUTPUT APPROPRIATE RESULT:
+"A + B = "  ...
+"A - B = "   ...
+"A * B = "   ...
+"A / B = "   ...
+"A**B = "   ...
+"A//B = "   ...
+"A%B = "  ...
+"""
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW1/hw_1_practical_task_2.py:16]: good job

File: Stanislav_Hryhorskiy/HW3/hw_3_practical_task_1.py [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+12. In the face of ambiguity, refuse the temptation to guess.
+13. There should be one-- and preferably only one --obvious way to do it.
+14. Although that way may not be obvious at first unless you're Dutch.
+15. Now is better than never.
+16. Although never is often better than *right* now.
+17. If the implementation is hard to explain, it's a bad idea.
+18. If the implementation is easy to explain, it may be a good idea.
+19. Namespaces are one honking great idea -- let's do more of those!
+"""
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW3/hw_3_practical_task_1.py:31]: good job

File: Stanislav_Hryhorskiy/HW3/hw_3_practical_task_2.py [hunk 4]
```diff
@@ -0,0 +1,32 @@
+"""
+*** Practical task 2 ***
+A four-digit natural number is specified:
+- find the product of the digits of this number;
+- write the number in reverse order;
+- in ascending order, you need to sort the numbers included in the given number.
+"""
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW3/hw_3_practical_task_2.py:8]: good job
and please look at this implementation:

number = int(input("Enter your number: "))

one_div, remainder = divmod(number, 1000)
two_div, remainder = divmod(remainder, 100)
three_div, four_div = divmod(remainder, 10)

#### Item 1: find of product of the digits of this number:
prod_of_dig = one_div * two_div * three_div * four_div
print("Product of the digits of this number:", prod_of_dig)

#### Item 2: write the number in reverse order:
str_number = str(number)[::-1]
print("Reverse of our number:", str_number)

#### Item 3: in ascending order, you need to sort the number included in the given number:
sort_number = ''.join(sorted(str(number)))
print("Ascending order of digits in the number:", sort_number)

File: Stanislav_Hryhorskiy/HW3/hw_3_practical_task_3.py [hunk 5]
```diff
@@ -0,0 +1,17 @@
+"""
+*** Practical task 3 ***
+Interchange the values of two variables
+without using the third variable.
+"""
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW3/hw_3_practical_task_3.py:6]: good job

File: Stanislav_Hryhorskiy/HW4/hw4_practical_task_0.py [hunk 6]
```diff
@@ -0,0 +1,28 @@
+"""
+*** Practical task 0 ***
+Write a script that will calculate the factorial
+of the entered number without using recursion.
+"""
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW4/hw4_practical_task_0.py]: good job

File: Stanislav_Hryhorskiy/HW1/hw_1_practical_task_1.py (hunk 1 above)
Comment (Stanislav-Hryhorskyi) [reply Stanislav_Hryhorskiy/HW1/hw_1_practical_task_1.py:12]: thank you!

File: Stanislav_Hryhorskiy/HW1/hw_1_practical_task_2.py (hunk 2 above)
Comment (Stanislav-Hryhorskyi) [reply Stanislav_Hryhorskiy/HW1/hw_1_practical_task_2.py:16]: thank you!

File: Stanislav_Hryhorskiy/HW3/hw_3_practical_task_1.py (hunk 3 above)
Comment (Stanislav-Hryhorskyi) [reply Stanislav_Hryhorskiy/HW3/hw_3_practical_task_1.py:31]: thank you!

File: Stanislav_Hryhorskiy/HW3/hw_3_practical_task_2.py (hunk 4 above)
Comment (Stanislav-Hryhorskyi) [reply Stanislav_Hryhorskiy/HW3/hw_3_practical_task_2.py:8]: thank you. pretty cool feature.

File: Stanislav_Hryhorskiy/HW3/hw_3_practical_task_3.py (hunk 5 above)
Comment (Stanislav-Hryhorskyi) [reply Stanislav_Hryhorskiy/HW3/hw_3_practical_task_3.py:6]: thank you.

File: Stanislav_Hryhorskiy/HW4/hw4_practical_task_0.py (hunk 6 above)
Comment (Stanislav-Hryhorskyi) [reply Stanislav_Hryhorskiy/HW4/hw4_practical_task_0.py]: thank you.

File: Stanislav_Hryhorskiy/HW4/hw_4_practical_task_1.py [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+Note: You can assume that the user will enter a valid input
+(a number for the temperature in Celsius).
+Example output:
+Enter the temperature in Celsius: 25
+25°C is equivalent to 77°F
+Example output (if the user enters a temperature below absolute zero):
+Enter the temperature in Celsius: -300
+Error: Temperature below absolute zero (-273.15°C)
+"""
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW4/hw_4_practical_task_1.py:18]: good job

File: Stanislav_Hryhorskiy/HW4/time_to_practice_4.py [hunk 8]
```diff
@@ -0,0 +1,31 @@
+"""
+Write a script, which of the two entered numbers will determine which of them is
+more and which is less. Take into account the case of equality of numbers.
+"""
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW4/time_to_practice_4.py:5]: good job

File: Stanislav_Hryhorskiy/HW5/hw5_practical_task_1.py [hunk 9]
```diff
@@ -0,0 +1,25 @@
+"""
+*** Practical task 1 ***
+Create a list that contains elements of integer type, then use
+the loop to change the type of these elements to a floating type.
+(Hint: use the built-in float() function).
+"""
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW5/hw5_practical_task_1.py:7]: good job

File: Stanislav_Hryhorskiy/HW5/hw5_practical_task_2.py [hunk 10]
```diff
@@ -0,0 +1,26 @@
+"""
+*** Practical task 2 ***
+Print Fibonacci numbers up to the entered number n, using cycles.
+(Sequence of Fibonacci numbers 0, 1, 1, 2, 3, 5, 8, 13, etc.)
+"""
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW5/hw5_practical_task_2.py:6]: good job

File: Stanislav_Hryhorskiy/HW5/hw5_practical_task_3.py [hunk 11]
```diff
@@ -0,0 +1,28 @@
+"""
+*** Practical task 3 ***
+Write a script that will calculate the factorial
+of the entered number without using recursion.
+"""
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW5/hw5_practical_task_3.py:6]: good job

File: Stanislav_Hryhorskiy/HW5/time_to_practice_5.py [hunk 12]
```diff
@@ -0,0 +1,46 @@
+"""
+Print all even numbers less than 100 (write two variants of the code:
+one using the while loop, and the other using the for loop).
+"""
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW5/time_to_practice_5.py:5]: good job

File: Stanislav_Hryhorskiy/HW6/hw6_practical_task_1.py [hunk 13]
```diff
@@ -0,0 +1,22 @@
+"""
+*** Practical task 1 ***
+In the range from 1 to 10 determine:
+• even numbers that are divisible by 2;
+• odd numbers, which are divisible by 3;
+• numbers that are not divisible by 2 and 3.
+"""
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW6/hw6_practical_task_1.py:8]: output your program:
In the range from 1 to 10:
        • even numbers that are divisible by 2: 2, 4, 6, 8, 10;
        • odd numbers, which are divisible by 3: 3, 6, 9;      
        • numbers that are not divisible by 2 and 3: 1, 5, 7.
6 is in first and second list. It is bad solution. Please try again do this task

File: Stanislav_Hryhorskiy/HW6/hw6_practical_task_2.py [hunk 14]
```diff
@@ -0,0 +1,16 @@
+"""
+*** Practical task 2 ***
+Write a script that checks the login that the user enters.
+If the login is "First", then greet the users. If the login is
+different, send an error message. (need to use loop while)
+"""
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW6/hw6_practical_task_2.py:7]: good job

File: Stanislav_Hryhorskiy/HW6/time_to_practice_6.py [hunk 15]
```diff
@@ -0,0 +1,27 @@
+"""
+Create a list of integers that are entered from the terminal
+and determine the maximum and minimum number among them.
+"""
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW6/time_to_practice_6.py:5]: good job

File: Stanislav_Hryhorskiy/HW6/hw6_practical_task_1.py (hunk 13 above)
Comment (Stanislav-Hryhorskyi) [reply Stanislav_Hryhorskiy/HW6/hw6_practical_task_1.py:8]: Thank you. I realized my mistake.

File: Stanislav_Hryhorskiy/HW7/hw7_practical_task_1.py [hunk 16]
```diff
@@ -0,0 +1,23 @@
+# *** Practical task 1 ***
+# Write a function that returns the largest number of two numbers
+# (use DocStrings documentation strings in the function).
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW7/hw7_practical_task_1.py:4]: good job

File: Stanislav_Hryhorskiy/HW7/hw7_practical_task_2.py [hunk 17]
```diff
@@ -0,0 +1,78 @@
+# *** Practical task 2 ***
+# Write a program that calculates the area of a rectangle,
+# triangle and circle (write three functions to calculate the area.
+# And call them in the main program depending on the user's choice).
+from math import pi
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW7/hw7_practical_task_2.py:5]: good job

File: Stanislav_Hryhorskiy/HW7/hw7_practical_task_3.py [hunk 18]
```diff
@@ -0,0 +1,21 @@
+# *** Practical task 3 ***
+# Write a function that calculates the number
+# of characters included in given string.
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW7/hw7_practical_task_3.py:4]: good job

File: Stanislav_Hryhorskiy/HW7/hw7_practical_tasks_codewars.py [hunk 19]
```diff
@@ -0,0 +1,141 @@
+# ***************************************************
+# I. Jenny's secret message
+# Jenny has written a function that returns a greeting for a user.
+# However, she's in love with Johnny, and would like to greet him
+# slightly different. She added a special case to her function,
+# but she made a mistake. Can you help her?
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW7/hw7_practical_tasks_codewars.py:7]: good job

File: Stanislav_Hryhorskiy/HW7/time_to_practice_7.py [hunk 20]
```diff
@@ -0,0 +1,18 @@
+# Write a function that returns
+# the arithmetic mean of any number of numbers.
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW7/time_to_practice_7.py:3]: good job

File: Stanislav_Hryhorskiy/HW8/hw8_practical_task_1/main.py [hunk 21]
```diff
@@ -0,0 +1,4 @@
+from utils import *
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW8/hw8_practical_task_1/main.py:1]: good job

File: Stanislav_Hryhorskiy/HW8/hw8_practical_task_3/function_module.py [hunk 22]
```diff
@@ -0,0 +1,36 @@
+from math import pi, pow
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW8/hw8_practical_task_3/function_module.py:1]: good job

File: Stanislav_Hryhorskiy/HW8/hw8_practical_task_3/task_3.py [hunk 23]
```diff
@@ -0,0 +1,46 @@
+# *** Practical task 3 ***
+# Write a program that calculates the area of a rectangle S=a*b,
+# the area of a triangle S=0.5*h*a, the area of a circle S=pi*r**2.
+# This module must be used in another module in which we ask the user
+# the area of which figure he wants to calculate.
+# (To perform the task, you need to import the math module, and from
+# it the pow() function and the value of the variable pi, and module,
+# which contains three functions for finding areas, into the main program.
+# The basic logic of the program is executed in the main module).
+
```
Comment (kolyasalubov) [inline Stanislav_Hryhorskiy/HW8/hw8_practical_task_3/task_3.py:10]: good job

**Pull request merged**